src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -115,4 +129,57 @@ public void nullValue() {
 
     Assert.assertEquals(actual, expected);
   }
+
+  @Test
+  public void getUserToNotifyQuestion() {
+    // Get list of users' IDs who follow an specific question.
+    HttpServletRequest request = mock(HttpServletRequest.class);
+    String typeOfNotification = "question";
+    int modifiedElementId = 2;
+
+    List<Integer> actual = Utility.getUsersToNotify(typeOfNotification, modifiedElementId, request);
+    List<Integer> expected = new ArrayList<>(List.of(4,6,7));
+
+    Assert.assertEquals(actual, expected);
+  }
+
+  @Test
+  public void getUserToNotifyNoType() {
+    // No type of notification included.
+    HttpServletRequest request = mock(HttpServletRequest.class);
+    String typeOfNotification = "";
+    int modifiedElementId = 2;
+
+    List<Integer> actual = Utility.getUsersToNotify(typeOfNotification, modifiedElementId, request);
+    List<Integer> expected = new ArrayList<>();
+    
+    Assert.assertEquals(actual, expected);
+  }
+
+  @Test
```

Comment:
nit: Add comment before this test.
`/** Tests for buildComment() function */`

Reply:
Done!